torch/onnx/symbolic_opset9.py

```diff
@@ -5579,6 +5584,11 @@ def min(g, self, other=None):
     def data(g, self):
         return self
 
+    @staticmethod
+    def layout(g, self):
```

Comment:
I am wondering if there is a way that we can settle these unused aten::op once for all, since they are missed pretty often.